### PR TITLE
[FIX]Fixes Unused-Variables warnings

### PR DESCRIPTION
**Pull request by shucon**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used CCExtractor.
- [x] I have used CCExtractor just a couple of times.
- [ ] I absolutely love CCExtractor, but have not contributed previously.
- [ ] I am an active contributor to CCExtractor.

---

Partially-Fixes https://github.com/CCExtractor/ccextractor/issues/780

### CODE REVIEW

File: src/lib_ccx/ccx_decoders_isdb.c [hunk 1]
```diff
@@ -1329,13 +1327,9 @@ int isdb_parse_data_group(void *codec_ctx,const uint8_t *buf, struct cc_subtitle
 	ISDBSubContext *ctx = codec_ctx;
 	const uint8_t *buf_pivot = buf;
 	int id = (*buf >> 2);
-	int version = (*buf & 2);
-	int link_number = 0;
-	int last_link_number = 0;
 	int group_size = 0;
```
Comment (cfsmp3) [inline src/lib_ccx/ccx_decoders_isdb.c]: Let's leave these ones, at least the version one. We might not be using in code, but it's useful when debugging - inspecting the variable and checking that the value is sane is a good way to see if things are going OK.

**Comment by shucon** [comment]
@cfsmp3 kindly review the changes.

**Comment by techfreakworm** [comment]
@shucon Just a polite ping for minor reminder. It would be good if you do not change the PR template. If you want to add any PR related info. Add it at last where `{Pull request content here}` is written.

File: src/lib_ccx/ccx_decoders_isdb.c [hunk 2]
```diff
@@ -1345,12 +1340,8 @@ int isdb_parse_data_group(void *codec_ctx,const uint8_t *buf, struct cc_subtitle
 		isdb_log("ISDB group B\n");
 	}
 
-	isdb_log("ISDB (Data group) version %d\n",version);
-
 	buf++;
-	link_number = *buf++;
-	last_link_number = *buf++;
-	isdb_log("ISDB (Data group) link_number %d last_link_number %d\n", link_number, last_link_number);
+	*buf++;
```
Comment (saurabhshri) [inline src/lib_ccx/ccx_decoders_isdb.c]: I see `*buf` incrementing twice..

File: src/lib_ccx/ccx_decoders_isdb.c [hunk 3]
```diff
@@ -1345,12 +1340,8 @@ int isdb_parse_data_group(void *codec_ctx,const uint8_t *buf, struct cc_subtitle
 		isdb_log("ISDB group B\n");
 	}
 
-	isdb_log("ISDB (Data group) version %d\n",version);
```
Comment (saurabhshri) [inline src/lib_ccx/ccx_decoders_isdb.c]: Any specific reason for deleting this?

Comment (shucon) [reply src/lib_ccx/ccx_decoders_isdb.c]: I deleted version variable so CLion deleted it automatically , as I have restored it I'll add this line again.

File: src/lib_ccx/ccx_decoders_isdb.c (hunk 2 above)
Comment (shucon) [reply src/lib_ccx/ccx_decoders_isdb.c]: Should I increment it twice ? I think there is no need to increment in twice.

**Pull request closed without merging**